app/models/sections/transportUnit/TransportUnitsAddToListModel.scala

```diff
@@ -35,23 +38,43 @@ object TransportUnitsAddToListModel extends Enumerable.Implicits {
     Yes, MoreToCome, NoMoreToCome
   )
 
-  def options(implicit messages: Messages): Seq[RadioItem] = {
+  def options(implicit messages: Messages, request: DataRequest[_], transportUnitSize: Int, transportUnits: Seq[SummaryList]): Seq[RadioItem] = {
     def radioItem(value: TransportUnitsAddToListModel, index: Int): RadioItem = RadioItem(
-      content = Text(messages(s"transportUnitsAddToList.${value.toString}")),
+      content = if (index == 1) {
+        Text(pluralSingular(s"transportUnitsAddToList.${value.toString}", transportUnitSize))
+      }
+      else Text(messages(s"transportUnitsAddToList.${value.toString}")),
       value = Some(value.toString),
       id = Some(s"value_$index")
     )
 
+    def transportUnitIncomplete(transportUnit: SummaryList): Boolean = {
+      if (transportUnit.rows.apply(0).value.content.asHtml.toString() == "Fixed transport installations") {
```

Comment:
Would it be possible to do these checks another way rather than on string matching of the description (Fixed transport installations, Not provided) because if we change those strings within _**messages**_ it wouldn't be clear that this code would also require changing resulting in unexpected behaviour?